cmake/modules/Distutils.cmake

```diff
@@ -63,7 +67,9 @@ function(distutils_install_cython_module name)
   install(CODE "
     set(ENV{CC} \"${PY_CC}\")
     set(ENV{LDSHARED} \"${PY_LDSHARED}\")
-    set(ENV{CPPFLAGS} \"-iquote${CMAKE_SOURCE_DIR}/src/include\")
+    set(ENV{CPPFLAGS} \"-iquote${CMAKE_SOURCE_DIR}/src/include
+                        -D'void0=dead_function\(void\)' \
```

Comment:
Um. Guessing. Could the backslash at the end of this line be causing problems with string parsing? Or, conversely, the lack of backslash on the previous line?  That said, isn't it odd that it didn't fail when #27067 was tested?

Reply:
@tserong the reason why #27067 didn't fail the test is that we are using cmake3 since #22896.